docs/integrations/sources/help-scout.md

```diff
@@ -0,0 +1,72 @@
+# Help Scout
+
+This page contains the setup guide and reference information for the Help Scout source connector.
+
+## Prerequisites
+
+1. Create a Help Scout account.
+2. Create an application and note the App ID and App Secret.
+3. Navigate to `https://secure.helpscout.net/authentication/authorizeClientApplication?client_id={application_id}&state={your_secret}`
```

Comment:
This isn't required - if you use the client-credentials flow, you can just use the app id as the client id, and app secret as the client secret in Airbyte

https://developer.helpscout.com/mailbox-api/overview/authentication/#client-credentials-flow